pages/news.html

```diff
@@ -87,7 +87,8 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
-            <p><strong>09/12/2021</strong> - Uyuni 2021.12 is now available! Key highlights: Salt a as bundle, aarch64 for openSUSE Leap 15.3, CentOS and clones clients, prometheus blackbox exporter and much more! Check the <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details.</p>
+            <p><strong>13/12/2021</strong> - With regard to the recent publication of the <a href="https://nvd.nist.gov/vuln/detail/CVE-2021-44228" target="_blank">Log4Shell vulnerability</a>,  we can <a href="here-link-to-message-at-the-list-archive" target="_blank">announce</a> that as to our current knowledge Uyuni installations are not affected.</p>
+            <p><strong>09/12/2021</strong> - Uyuni 2021.12 is now available! Key highlights: Salt as a bundle, aarch64 for openSUSE Leap 15.3, CentOS and clones clients, prometheus blackbox exporter and much more! Check the <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details.</p>
```

Comment:
This line should not get any changes. Maybe you changed spaces to tabs, or tabs to spaces?

Reply:
I fixed a typo reported by @vzhestkov (in a separate commit): "Salt a as bundle" vs. "Salt as a bundle"